packages/tpetra/core/src/Tpetra_Import_def.hpp

```diff
@@ -1064,205 +1077,224 @@ namespace Tpetra {
 
   template <class LocalOrdinal, class GlobalOrdinal, class Node>
   void Import<LocalOrdinal,GlobalOrdinal,Node>::
-  setupExport (Teuchos::Array<GlobalOrdinal>& remoteGIDs,
-               bool useRemotePIDs,
-               Teuchos::Array<int>& userRemotePIDs)
+  setupExport(Teuchos::Array<GlobalOrdinal>& remoteGIDs,
```

Comment:
It's a bit hard to tell what happened in this routine, given the whitespace and formatting changes.